packages/template-tab-navigation-ts/app/_app-common.scss

```diff
@@ -25,4 +25,20 @@
 //Action bar
 ActionBar .action-bar-title {
     horizontal-align: center;
-}
\ No newline at end of file
+}
+
+TabStripItem Label {
```

Comment:
We can do this whole section with the `TabStripItem` only since the `color` property is inherited.

Reply:
Actually left it for consistency as platform-specific css needs to set `text-transform` and (as discussed previously) this is not inherited css property due to historic reasons so we need to use TabStripItem Label { ... } selector there.